ctapipe/core/selector.py

```diff
@@ -2,11 +2,17 @@
 Data Quality selection
 """
 
-__all__ = ["Selector"]
+__all__ = ["Selector", "SelectionFunctionError"]
+
+from collections import Callable
```

Comment:
`from collections.abc`, importing directly from collections raises a warning in 3.7 and an error in 3.8

Reply:
done